packages/web-components/src/components/ic-page-header/ic-page-header.css

```diff
@@ -92,6 +92,12 @@ header.tabs {
 }
 
 @media screen and (max-width: 576px) {
+  ::slotted(ic-text-field) {
+    --input-width: 100%;
+
```

Comment:
Do we need this extra line?

Reply:
No that was a mistake, I'll remove that now